posix-configs/ocpoc/px4.config

```diff
@@ -0,0 +1,37 @@
+ver all
+uorb start
+
+param select /home/root/rootfs/eeprom/parameters
+param load
+param set SYS_AUTOSTART 4001
+param set MAV_BROADCAST 1
+sleep 1
+param set MAV_TYPE 2
+param set MAV_SYS_ID 1
+param set BAT_CNT_V_VOLT 18.857142857
+param set BAT_V_DIV 0.000244141
+
+df_mpu9250_wrapper start_without_mag -R 10
+df_hmc5883_wrapper start
+df_ms5611_wrapper start
+rgbled start -b /dev/i2c-1

```

Comment:
I think I remember seeing the same thing, that this doesn't actually do anything, and I probably just overlooked taking this -b option out from the px4.config file.